laravel/composer.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
This is the file that defines the PHP dependencies. You typically shouldn't need to hand modify this, instead you can use the composer commandline.

```
vagrant ssh
cd /vagrant/laravel
composer require barryvdh/laravel-ide-helper
```